### PR TITLE
5 make runnable on multiple datasets

### PR DESCRIPTION
**Pull request by dimalvovs**
This is a code review template, it will be displayed after each pull request. We do code reviews because we care about quality of our codebase and because we care about long-term development goals. We do not do code reviews to blame the ones who contribute. There is no seniority in this process and there is no perfect code, we just want to avoid storing code that is not OK.

# For the submitter
- [x]  The change is sufficiently small

# For the approver
- [x] OK, code does what is intended
- [ ] its' OK to do it that way
- [ ] it's OK to read and comprehend the code
- [ ] tests, documentation, style are OK

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -24,10 +24,16 @@ process PREPROCESS {
   container 'docker.io/satijalab/seurat:5.0.0'
 
   input:
-      path data 
+      tuple val(sample), path(data)
```
Comment (atuldeshpande) [inline main.nf:27]: Is `sample` the sample name from the samplesheet.csv? How do we use it? Does the path() return the path of the specific sample?

File: main.nf [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // usage
-// nextflow run main.nf --data ~/Documents/data/breastcancer -w wd -resume -profile docker
+// nextflow run main.nf --data ./samplesheet -w wd -resume -profile docker
```
[the remark this answers is not in the document]
Comment (dimalvovs) [reply main.nf:2]: makes sense, let's do that!

File: main.nf [hunk 3]
```diff
@@ -1,5 +1,5 @@
 // usage
-// nextflow run main.nf --data ./samplesheet -w wd -resume -profile docker
+// nextflow run main.nf --input ./samplesheet -w wd -resume -profile docker
```
Comment (atuldeshpande) [inline main.nf:2]: Awesome, thanks!

**Review by atuldeshpande — APPROVED** [review]
Awesome!

**Pull request merged**